python/packages/jumpstarter-driver-gptp/README.md

```
@@ -0,0 +1,282 @@
+| ----------------- | ---------------------------- | ----------------------------- |
+| Transport         | Layer 2 only                 | L2, UDPv4, UDPv6             |
+| Timestamping      | Hardware required             | HW or software               |
+| Accuracy          | Sub-microsecond              | Sub-microsecond to ms         |
+| Use case          | Automotive, industrial TSN   | General purpose               |
+| Profile setting   | `profile: gptp`              | `profile: default`            |
+
+### Port State Machine
+
+PTP ports transition through these states:
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)